packages/editor-ui/src/modules/nodeTypes.ts

```diff
@@ -58,8 +58,7 @@ const module: Module<INodeTypesState, IRootState> = {
 	},
 	mutations: {
 		setNodeTypes(state, newNodeTypes: INodeTypeDescription[]) {
-			// tslint:disable-next-line:no-any
-			const nodeTypes = newNodeTypes.reduce((acc: any, newNodeType) => {
+			const nodeTypes = newNodeTypes.reduce((acc: Record<string, Record<string, INodeTypeDescription>>, newNodeType) => {
```

Comment:
Nit: The type for the accumulator is provided [as a generic](https://github.com/microsoft/TypeScript/blob/afee8bf9682dfb3ed927fdc7756d96a19851b811/lib/lib.es5.d.ts#L1386) to `reduce`

Reply:
Today I learned 😃